src/component/base/methods.js

```diff
@@ -51,6 +52,39 @@ export default {
     enumerable: true,
     configurable: false,
   },
+  $bubbleInput: {
```

Comment:
I'm not sure I like the name `$bubbleInput` ...

Maybe it should just be `$input`? Also, would it then make sense that you call this on the actual component that you want to apply the `$input` on, instead of automatically on the parent? That would make it more flexible and work pretty much the same in terms of logic as `$focus()` ..